src/Data/README.md

```diff
@@ -0,0 +1,46 @@
+NBB.Data
+===============
+
+In Martin Fowler’s PoEAA, the Repository pattern is described as:
+> Mediates between the domain and data mapping layers using a collection-like interface for accessing domain objects.
+
+
+This collection of packages focus on formalizing data access abstractions and offer some unambitious implementations.
```

Comment:
"This collection ... focuses... and offers..."
"These packages are"
"CRUD" acronym